cmd/cluster/core/create.go

```diff
@@ -597,6 +601,15 @@ func (opts *RawCreateOptions) Validate(ctx context.Context) (*ValidatedCreateOpt
 		} else if !apierrors.IsNotFound(err) {
 			return nil, fmt.Errorf("hostedcluster doesn't exist validation failed with error: %w", err)
 		}
+
+		// Validate multi-arch aspects
+		kc, err := hyperutil.GetKubeClientSet()
+		if err != nil {
+			return nil, err
```

Comment:
Generally it's good practice to wrap errors in additional context rather than returning a naked error. It seems like you've done this in some places but not all right now?
```suggestion
			return nil, fmt.Errorf("could not retrieve kube clientset: %w", err)
```